File: src/Do-ho/dataStructor/pgs_위장/pgs_위장.py
def solution(clothes):
    answer = 1
    hash = {}
    for item in clothes:
        try: hash[item[1]] += 1
        except: hash[item[1]] = 1 + 1
    
    for val in list(hash.values()): answer *= val
    
    return answer - 1

File: src/Do-ho/dataStructor/pgs_위장/pgs_위장.js
const solution = clothes => {
    const clothesObject = clothes.reduce((acc, [name, kind]) => {
        acc.hasOwnProperty(kind)? acc[kind] += 1 : acc[kind] = 1;
        return acc;
    }, new Object());
    
    return Object.values(clothesObject).reduce((acc, val) => acc *= val + 1, 1) - 1;
}